addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -267,7 +267,7 @@ function factory(dependencies) {
                 }
                 // (re)open chat on receiving new message
                 if (channel.channel_type !== 'channel' && !this.env.messaging.device.isMobile) {
-                    this.env.messaging.chatWindowManager.openThread(channel);
+                    this.env.messaging.chatWindowManager.openThread(channel, { isFolded: channel.chatWindowIsFolded });
```

Comment:
The `openThread` should not be called if there is already a chat window then, because it will do nothing.

It would be better to simply not call it in that case.